src/main/kotlin/blackjack/controller/Game.kt

```diff
@@ -0,0 +1,70 @@
+package blackjack.controller
+
+import blackjack.model.Dealer
+import blackjack.model.Deck
+import blackjack.model.Player
+import blackjack.view.InputView
+import blackjack.view.OutputView
+
+class Game(
+    private val inputView: InputView,
+    private val outputView: OutputView,
+    private val deck: Deck,
+    private val dealer: Dealer,
+) {
+    fun startGame() {
+        val playersName = inputView.askPlayerNames()
+        val players = playersName.map { Player(it) }
+        dealCardsToDealer()
+        dealCardsToPlayer(players)
+        outputView.displayInitialCards(dealer, players)
+        askPlayersToDraw(players)
+        dealerDraws()
+        outputView.displayCardsWithTotalValue(dealer, players)
+        compareFinalCards(players)
+        outputView.displayFinalResults(dealer, players)
+    }
+
+    fun dealCardsToDealer() {
+        repeat(2) {
+            dealer.drawCard(deck.giveCard())
+        }
+    }
```

Comment:
```suggestion
    private fun dealCardsToDealer() {
        repeat(2) {
            dealer.drawCard(deck.giveCard())
        }
    }
```

Let's keep things private if they aren't needed in outside world 🤫 .
Please check the whole project about this part.

Reply:
Thanks for the suggestion. Making functions `private` when they’re not used externally is a great practice to encapsulate implementation details and reduce unintended usage.

I have reviewed the entire project and marked all such internal helper methods `private` to keep the API clean and focused.I hope that I didn't miss any.